crates/egui/src/ui.rs

```diff
@@ -2811,6 +2884,10 @@ impl Ui {
 #[cfg(debug_assertions)]
 impl Drop for Ui {
     fn drop(&mut self) {
+        if self.should_interact_bg_on_drop {
+            #[allow(deprecated)]
+            self.interact_bg();
```

Comment:
a why-comment would be useful here

```suggestion
            // Register our final `min_rect`
            self.interact_bg();
```